roles/jd-server/config-examples/jds-config-hosted-example.toml

```diff
@@ -18,8 +18,8 @@ coinbase_outputs = [
 # SRI Pool JD config
 listen_jd_address = "0.0.0.0:34264"
 # RPC config for mempool (it can be also the same TP if correctly configured)
-core_rpc_url =  "http://127.0.0.1"
```

Comment:
@GitGab19 why was this `127.0.0.1`?

Reply:
Because `jds-config-hosted-example.toml` has been always used just by us on our VPS. We don't expose our RPCs to externals. So this file was only meant to be used by us.
But we can change this, and start serving RPCs to internet, so that everybody can spin up a Pool, a JDS, and everything using our hosted TP. What do you think?